interface/main/tabs/main.php

```diff
@@ -274,12 +274,12 @@ function setupI18n(lang_id) {
 <script type="text/javascript">
     <?php if (!empty($_SESSION['frame1url']) && !empty($_SESSION['frame1target'])) { ?>
         // Use session variables and tabStatus object to set up initial/default first tab
-        app_view_model.application_data.tabs.tabsList.push(new tabStatus(<?php echo xlj("Loading"); ?> + "...",<?php echo json_encode("../".$_SESSION['frame1url']); ?>,<?php echo json_encode($_SESSION['frame1target']); ?>,true,true,false));
+        app_view_model.application_data.tabs.tabsList.push(new tabStatus(<?php echo xlj("Loading"); ?> + " <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",<?php echo json_encode("../".$_SESSION['frame1url']); ?>,<?php echo json_encode($_SESSION['frame1target']); ?>,<?php echo json_encode($_SESSION['frame1label']); ?>,true,true,false));
```

Comment:
Same as previous comment
`<a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>`
to
`<i class='fa fa-spinner fa-pulse fa-sm'></i>`

Reply:
to fix the Loading undefined:
app_view_model.application_data.tabs.tabsList.push(new tabStatus(<?php echo xlj("Loading"); ?> + " <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",<?php echo json_encode("../".$_SESSION['frame1url']); ?>,<?php echo json_encode($_SESSION['frame1target']); ?>,<?php echo xlj("Loading"); ?> + " " + <?php echo json_encode($_SESSION['frame1label']); ?>,true,true,false));

Reply:
That worked like a charm 😀 🎉